images/ansible-operator/Pipfile

```diff
@@ -6,9 +6,9 @@ name = "pypi"
 [packages]
 ansible-runner = "~=2.3.3"
 ansible-runner-http = "~=1.0.0"
-ansible-core = "~=2.15.0"
+ansible-core = "~=2.15.9"
 urllib3 = "~=1.26.17"
-kubernetes = "==26.1.0"
+kubernetes = "==29.0.0"
```

Comment:
Is the version label correct?

Reply:
It's similar to what is present upstream. 

https://github.com/operator-framework/ansible-operator-plugins/blob/release-v1.35/images/ansible-operator/Pipfile#L11

Reply:
It is better to assess if it makes sense to use v30.1.0 based on the release changelog and [compatibilty matrix](https://github.com/kubernetes-client/python/tree/v30.1.0?tab=readme-ov-file#compatibility).

Reply:
I think it should happen upstream first, and we can then pull into downstream.